doc/user_guide/model.rst

```diff
@@ -247,7 +277,7 @@ index in the model.
 .. versionadded:: 0.8.1 :py:attr:`components` attribute
 
 In addition, the components can be accessed in the
-:py:attr:`~.model.Model.components` `Model` attribute. This is specially
+:py:attr:`~.models.model1D.Model1D.components` `Model` attribute. This is specially
```

Comment:
Any reason why the `components` module is not defined for the `BaseModel`? Of course if just copy-pasting does not work, let me know and we'll add that to the list of things not yet supported for `Model2D`